cmd/sync-tags/gomod.go

```diff
@@ -55,30 +55,46 @@ func updateGomodWithTaggedDependencies(tag string, depsRepo []string) (bool, err
 		rev := commit.String()
 		pseudoVersion := fmt.Sprintf("v0.0.0-%s-%s", commitTime.UTC().Format("20060102150405"), rev[:12])
 
+		downloadCommand := exec.Command("go", "mod", "download")
```

Comment:
will this always pick the right go version?

Reply:
Not always, but it _could_ if the pseudo version doesn't change. Added a comment.